chatgpt_automation/chatgpt_client.py

```diff
@@ -132,7 +132,14 @@ def interact(self, question : str):
     def reset_thread(self):
         """the conversation is refreshed"""
         self.browser.find_element(By.XPATH, self.reset_xq).click()
-
+    
+    def check_if_request_limit_exceeded(self):
+        elements = self.browser.find_elements(By.XPATH,'//div[@class="py-2 px-3 border text-gray-600 rounded-md text-sm dark:text-gray-100 border-red-500 bg-red-500/10"]')
```

Comment:
This line can fail easily since the class definition is way too specific, there should be something else to identify the check rate limiting

Reply:
I'm sorry I can't find another solution. You're totally free to modify it or even remove it from the code if you want.